### PR TITLE
Increased stack size

### PR DESCRIPTION
**Pull request by ivan-egorov42**
According to #1383

### CODE REVIEW

**Comment by ivan-egorov42** [comment]
@Graur pls, have a look.

File: eo-runtime/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@ SOFTWARE.
   <description>EO runtime library</description>
   <properties>
     <!--  This is required for later correct replacement of argline  -->
-    <argLine/>
+    <stack-size>256M</stack-size>
```
Comment (volodya-lombrozo) [inline eo-runtime/pom.xml:38]: I think it's too much. I've tried locally and tests become pass with:
```xml
<stack-size>2M</stack-size>
```
As far as I know, the default value is `1M`

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@ivan-egorov42 We can reduce the stack size

File: eo-runtime/pom.xml (hunk 1 above)
Comment (ivan-egorov42) [reply eo-runtime/pom.xml:38]: @volodya-lombrozo yes, it is. Maybe more size(256) can be needed in the future?

Comment (ivan-egorov42) [reply eo-runtime/pom.xml:38]: @volodya-lombrozo well, I think the same. Let it be.

**Review by volodya-lombrozo — APPROVED** [review]
@ivan-egorov42 looks good to me. @yegor256 merge, please.

**Comment by volodya-lombrozo** [comment]
@yegor256 reminder

File: eo-runtime/pom.xml [hunk 2]
```diff
@@ -96,6 +96,7 @@ SOFTWARE.
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <argLine>@{argLine} -Duser.language=ru -Duser.region=RU</argLine>
+          <argLine>-Xss${stack-size}</argLine>
```
Comment (yegor256) [inline eo-runtime/pom.xml:99]: @ivan-egorov42 see https://stackoverflow.com/questions/23190107/cannot-use-jacoco-jvm-args-and-surefire-jvm-args-together-in-maven This `@{argline}` here is for a reason.

**Comment by volodya-lombrozo** [comment]
@ivan-egorov42 you can update `master` branch and see how we add `args` for `eo-maven-plugin`. I think we need the same for `eo-runtime`

**Comment by volodya-lombrozo** [comment]
@ivan-egorov42 do you have any progress here? It's a quite important issue that breaks lots of CI/CD pipelines.

**Comment by Graur** [comment]
@ivan-egorov42 What's up with this?

**Comment by Graur** [comment]
closed

**Pull request closed without merging**